thirtyfour/src/session/mod.rs

```diff
@@ -1,8 +1,66 @@
-/// Code for starting a new session.
-pub mod create;
-/// The underlying session handle.
-pub mod handle;
-/// HTTP helpers for WebDriver commands.
-pub mod http;
-/// Helper for values returned from scripts.
-pub mod scriptret;
+/// Code for starting a new session.
+pub mod create;
+/// The underlying session handle.
+pub mod handle;
+/// HTTP helpers for WebDriver commands.
+pub mod http;
+/// Helper for values returned from scripts.
+pub mod scriptret;
+
+mod sealed {
+    use std::borrow::Cow;
+    use std::rc::Rc;
+    use std::sync::Arc;
+
+    pub trait IntoTransfer {
```

Comment:
Transfer, because were gonna transfer it to the other management task

IntoArcStr would be confusing because why not use Into<Arc<str>> which I didn't use because many of the types added to the impl don't implement Into<Arc<str>>

Reply:
if you have m=name suggestions I'm up for it I tried to come up with a good name for a good 10 minutes